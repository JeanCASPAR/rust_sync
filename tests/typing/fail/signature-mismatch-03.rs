use rust_sync::sync;

sync! {
    #![pass(1)]

    node oui() = (a)
    where
        a : int = 3;

    node non(b : bool) = ()
    where
        o : int = oui(b);
}

fn main() {}
