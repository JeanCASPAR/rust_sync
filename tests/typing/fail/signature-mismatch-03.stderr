error: node `oui` takes 0 arguments, but 1 was provided
  --> tests/typing/fail/signature-mismatch-03.rs:12:19
   |
12 |         o : int = oui(b);
   |                   ^^^^^^
